### DISCUSSION TITLE
How Do I Display Webpages In My Obsidian Vault?

### DISCUSSION THREAD

**Discussion by VepryB** [BryanJenksCommunity/FAQ#16] [Q&A] [answered]
### Question

What is your question?
- how do I display websites in my obsidian vault?
- Is it only available for obsidian publish?

I was looking at your obsidian publish and I noticed that you can display websites (?). Example: In https://publish.obsidian.md/bryan-jenks/VimWiki , you can publish https://blog.mague.com/?p=602 .

I can scroll down and click at the link to another page!

Before asking this question, I tried ![]() which probably only works with image? Ahahaha

---

By filing a question to this repo, I promise that

-   [x] I have searched open/closed issues on this repo for my answer already

I understand that my issue may be closed if I don't fulfill my promises.

[a comment by an automated account is not shown]

**Comment by tallguyjenks** [comment, marked as answer]
iframes! 🙂 

the exact content of that note is this:

```
[Getting started with vimwiki](https://blog.mague.com/?p=602)

<iframe width="560" height="700" src="https://blog.mague.com/?p=602"></iframe>

[vimwiki cheatsheet](http://thedarnedestthing.com/vimwiki%20cheatsheet)

<iframe width="560" height="700" src="http://thedarnedestthing.com/vimwiki%20cheatsheet"></iframe>
```

It is not limited to publish users, you can do this in your local vault as long as you have an internet connection for the iframe to fetch the webpage :) 

Copy and paste the contents into a note in your vault then go to preview mode ✨
